src/database/mutations/ScheduledItem.ts

```diff
@@ -44,6 +45,23 @@ export async function createScheduledItem(
   });
 }
 
+/**
+ * Create (import) a scheduled item for a given approved item
+ * @param db
+ * @param data
+ */
+export async function importScheduledItem(
+  db: PrismaClient,
+  data: ImportScheduledItemInput
+): Promise<ScheduledItem> {
+  return db.scheduledItem.create({
+    data,
+    include: {
```

Comment:
as we're returning the approved item as well, do we need this `include`?

Reply:
I was battling with types and took the path of least resistance, I will revisit

Reply:
We need this one for the snowplow event. We could rework that but that would increase the scope of this work by a lot